content/en/docs/chart_best_practices/dependencies.md

```diff
@@ -35,6 +35,19 @@ File URLs (`file://...`) are considered a "special case" for charts that are
 assembled by a fixed deployment pipeline. Charts that use `file://` are not
 allowed in the official Helm repository.
 
+#### Experimental support for Charts hosted on OCI registries
+
+If you have [enabled experimental OCI support](/docs/registries/), you can specify
+an OCI reference (`oci://registry/group/image:tag`) for the repository URL.
```

Comment:
The language through this and the linked doc make a lot of assumptions about what the reader knows and how they treat helm charts as an image in the registries. I'm not sure if it should be in this PR but we need something in a follow-up that helps someone who isn't an insider understand this.

Reply:
Yep, this is a good point. I do agree this could be more clear, especially with this being under `chart_best_practices`. I'm hesitant to add details about what an OCI registry _is_ here, that's out of scope. Happy to discuss approaches to this 🤔

Reply:
I think this change is good, if anything we need more background in registries page (i can add)

Reply:
opened https://github.com/helm/helm-www/issues/637